it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/uaControllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -286,23 +378,27 @@ class CheckYourAnswersControllerISpec extends AuditEnabledSpecHelper
         ))
 
         stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
-        stubRetrieveUtr(testJourneyId)(OK, testCtutrJson)
-        stubRetrievePostcode(testJourneyId)(OK, testOfficePostcode)
-        stubValidateUnincorporatedAssociationDetails(testCtutr, testOfficePostcode)(BAD_REQUEST, Json.obj("code" -> "NOT_FOUND",
-          "reason" -> "The back end has indicated that CT UTR cannot be returned"
-        ))
-        stubStoreIdentifiersMatch(testJourneyId, DetailsNotFoundKey)(OK)
+        stubRetrieveUtr(testJourneyId)(NOT_FOUND)
+        stubRetrievePostcode(testJourneyId)(NOT_FOUND)
+
+        stubStoreIdentifiersMatch(testJourneyId, UnMatchableKey)(OK)
+
+        stubStoreBusinessVerificationStatus(testJourneyId, expBody = testVerificationStatusJson(verificationStatusValue = "NOT_ENOUGH_INFORMATION_TO_CALL_BV"))(OK)
+        stubStoreRegistrationStatus(testJourneyId, RegistrationNotCalled)(OK)
+
         stubRetrieveEntityDetails(testJourneyId)(OK, testUAJourneyDataJsonNotFound)
```

Comment:
Again, I don't this journey data is correct, because it contains a utr and postcode.